static/css/external-site.less

```diff
@@ -0,0 +1,45 @@
+/* ==========================================================================
+   cfgov-refresh
+   Third-Party links pages at /external-site/.
+   ========================================================================== */
+
+.external-site_container {
+    .content_bar {
+        background: @gray-50;
+        height: 1px;
+    }
+
+    .padded-container {
+        background-color: @white;
+    }
+
+    .webfont-regular();
```

Comment:
This should be at the top since it's for the entire `.external-site_container` block.